app/src/main/java/com/example/advanceduiapp/ui/pager/PagerViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.advanceduiapp.ui.pager
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.advanceduiapp.R
+import com.example.advanceduiapp.data.PageData
+
+class PagerViewModel: ViewModel() {
+    val showMultipleViewRecyclerFragment = MutableLiveData<Boolean>(false)
```

Comment:
showMultipleViewRecyclerFragment should be immutable publicly. The state of the showMultipleViewRecyclerFragment should be change only from the viewModel